### DISCUSSION TITLE
received a message : Prevented user 842900455 to interact.. is now safe

### DISCUSSION THREAD

**Discussion by Asifadb** [beeb/pancaketrade#53] [General]
i received a message Prevented user 842900455 to interact
are my funds safe?

**Comment by beeb** [comment]
Yes your funds are safe. We cannot prevent people from trying to interact with any telegram bot. But pancaketrade ensures that they cannot call any command. This is why you provide your user ID in the config file.

**Reply by Asifadb** [reply]
Thank you very much for your reply.. i didnt expect fast reply.. i already transferred funds out of fear.. you are a savior.. once again thank you for your bot/ service... its an honor to interact with a developer like you..

**Comment by beeb** [comment]
If you get too many of these warnings, you should probably create a new bot token and update your config file. Make the bot username as random as you can.

**Reply by Asifadb** [reply]
Thank you
